apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -211,6 +218,87 @@ export class TeacherFeedback extends Component {
     return feedbackUnchanged;
   };
 
+  getFriendlyDate = feedbackSeen => {
+    const today = new Date();
+    const dateFeedbackSeen = new Date(feedbackSeen);
+
+    //javascript months begin counting at 0
+    const [todayM, todayD, todayY] = [
+      today.getMonth() + 1,
+      today.getDate(),
+      today.getFullYear()
+    ];
+
+    const [m, d, y] = [
+      dateFeedbackSeen.getMonth() + 1,
+      dateFeedbackSeen.getDate(),
+      dateFeedbackSeen.getFullYear()
+    ];
+    if (todayM === m && todayY === y) {
+      if (todayD === d) {
+        return 'today';
+      } else if (todayD === d + 1) {
+        return 'yesterday';
+      }
+    } else {
+      const feedbackYear = (y + '').substr(2, 2);
+      return `${m}/${d}/${feedbackYear}`;
+    }
+  };
+
+  //sub-render method invoked by getFeedbackText()
+  displayFeedbackMessage = ([style, message, studentSeen, time]) => {
+    return (
+      <div style={style} id="ui-test-feedback-time">
+        {studentSeen && (
+          <FontAwesome
+            icon="check"
+            className="fa-check"
+            style={styles.checkboxIcon}
+          />
+        )}
+        {`${message} `}
+        {time && <strong>{time}</strong>}
+      </div>
+    );
+  };
+
+  getFeedbackText = (latestFeedback, studentSawFeedback) => {
+    let timeStyle;
+    let time;
+    let feedbackMessage;
+    let currentTeacherStudentSeen = false;
+
+    //Student view indicates when feedback was last updated
+    if (this.props.viewAs === ViewType.Student) {
+      timeStyle = styles.timeStudent;
+      feedbackMessage = i18n.lastUpdated();
```

Comment:
Here and below: Rather than concatenating the string later, one possibility is to pass the time directly into the i18n message. example:
```
time = moment.min(moment(), moment(latestFeedback.created_at)).fromNow();
feedbackMessage = i18n.lastUpdated({time});
```
This format will make translations smoother.

Reply:
Got it. I pulled the times out of the i18n file since Amanda wants them to be bold, but that may just apply to the teacher view. I will pass `time` when viewing as student to the i18n file, and I’ll create separate entries for “today” and “yesterday” so they are internationalized as well.